src/Http/Controllers/Character/ContractsController.php

```diff
@@ -174,46 +174,22 @@ public function getContractsData(int $character_id)
                 return false;
             })
             ->filterColumn('issuer_id', function ($query, $keyword) {
-                $resolved_ids = UniverseName::where('name', 'like', '%' . $keyword . '%')
-                    ->get()
-                    ->map(function ($resolved_id) {
-                        return $resolved_id->entity_id;
-                    });
-                $character_info_ids = CharacterInfo::where('name', 'like', '%' . $keyword . '%')
-                    ->get()
-                    ->map(function ($character_info) {
-                        return $character_info->character_id;
-                    });
-
-                $query->whereIn('a.assignee_id', array_merge($resolved_ids->toArray(), $character_info_ids->toArray()));
+
+                $resolved_ids = $this->getIdsForNames($keyword);
+
+                $query->whereIn('a.assignee_id', $resolved_ids->toArray());
```

Comment:
Lets not alloc a new var here and instead just inline `$this->getIdsForNames($keyword)->toArray();`. Same would apply for the rest of the usage of this trait method going down.